src/compute/src/logging/timely.rs

```diff
@@ -55,26 +53,29 @@ pub(super) fn construct<A: Allocate>(
     event_queue: EventQueue<Vec<(Duration, TimelyEvent)>>,
     shared_state: Rc<RefCell<SharedLoggingState>>,
 ) -> BTreeMap<LogVariant, LogCollection> {
-    let logging_interval_ms = std::cmp::max(1, config.interval.as_millis());
+    let logging_interval_ms =
+        u64::try_from(std::cmp::max(1, config.interval.as_millis())).expect("must fit");
```

Comment:
What made this change necessary? This variable is only used in `DemuxHandler::ts` and there it looks like the types line up better if this stays an `u128`.

Reply:
Yup, I'll remove that change.